readme.md

```diff
@@ -230,6 +257,23 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
+var andy = Instructor.findOne({name: 'Andy'}, function(err, person){
+  if (err) {
+    console.log('No person found by that name')
+  } else {
+    console.log('The name is ' + person.name)
+  }
+})
+
+andy.wishlistItems.create({description: 'Resin Laying Deer Figurine, Gold'}, function(err, data){
+  if (err) {
+    console.log('error creating new wishlist item')
+  } else {
+    console.log('Successfully created ' + data + 'wishlist item')
+  }
+}) //camelcase for JS?
+
+// not sure how right this is - I also feel like there's a way to save the wishlist items at the
+// same time in the callback to findOne?
 
```

Comment:
:+1: This is valid!

You could add something like this to your `andy` variable to eliminate the second step...
```
person.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
person.save();
```